### ISSUE TITLE
Add a feature to create networks

### ISSUE THREAD

**Issue by TsutomuNakamura** [tsuna-server/build-openstack-deployment#12]

**Comment by TsutomuNakamura** [comment]
Examples if we use a command "openstack network create".

```
* External network with Linux Bridge
  openstack network create --provider-network-type ${provider_network_type} --provider-physical-network ${provider_physical_network} --external ${network_name}

* External network with OVN
  openstack network create --provider-network-type #{provider_network_type} --provider-physical-network ${provider_physical_network} --external ${network_name}

* Internal network with Linux Bridge
  openstack network create --mtu ${mtu} ${network_name}

* Internal network with OVN (--provider-network-type and --provider_segment may not be required)
  openstack network create --mtu ${mtu} --provider-network-type ${network_type} --provider-segment ${provider_segment} ${network_name}
```

**Comment by TsutomuNakamura** [comment]
This issue is already solved at ed423eeb9b36b563eba5d2fa36f6dde752df911a.
